frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -160,6 +166,7 @@ class Recipe extends Component {
             }
           </ul>
         </div>
+        { comments ? <RecipeComments comments={comments} recipeId={recipeId} user={user} /> : null }
```

Comment:
I might use `!isNull(comments)` (with `isNull` coming from lodash). Right now, this has the right behavior, but the intent is a little murky (especially given that the truthiness of `[]` varies from language to language).

Reply:
Would we need to create a separated if statement in order to employ isNull, or can we do it within the ternary?